pom.xml

```diff
@@ -24,7 +24,7 @@
 
   <properties>
     <rhba.images.version.prefix>77</rhba.images.version.prefix>
-    <rhba.images.branch>master</rhba.images.branch>
+    <rhba.images.branch>7.7.x</rhba.images.branch>
```

Comment:
please set it back to master as this PR is to master branch, you probably changed that for testing purposes

Reply:
My bad. Reverted